File: Logic Building Assignments/Logic Building Assignment 3/Assignment3_2.c
//  2. Write a program which accept number from user and print factors
//     of that number.

#include<stdio.h>

void DisplayFactor(int iNo)
{
    int iCnt = 0;

    if(iNo <= 0)
    {
        return;
    }

    for(iCnt = 1; iCnt <= (iNo/2); iCnt++)
    {
        if((iNo % iCnt) == 0)
        {
            printf("%d\t",iCnt);
        }
    }
}

int main()
{
    int iValue = 0;

    printf("Enter your number : ");
    scanf("%d",&iValue);

    DisplayFactor(iValue);

    return 0;
}

/*

    Output:
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>gcc Assignment3_2.c

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>a
    Enter your number : 24
    1       2       3       4       6       8       12

*/

File: Python Assignments/Python Assignment 3/Assignment3_2.py
#   2. Write a program which accept N numbers from user and store it into
#      List. Return Maximum number from that List.

def MaxElement(Data):
    Max = Data[0]
    for i in Data:
        if Max<i:
            Max = i
        else:
            continue
    return Max

def main():
    Size = int(input("How many elements do you want in the list : "))
    Arr = list()

    print("Enter a list elements")
    for i in range(Size):
        No = int(input())
        Arr.append(No)
    print("Your Entered List Elements are : ", Arr)

    Max = MaxElement(Arr)
    print("Maximum element of the given list is : ",Max)

if __name__ == "__main__":
    main()

#   Output:
#   D:\Python\Marvellous Python\Python Assignments\Python Assignment 3>Python Assignment3_2.py
#   How many elements do you want in the list : 7
#   Enter a list elements
#   13
#   5
#   45
#   7
#   4
#   56
#   34
#   Your Entered List Elements are :  [13, 5, 45, 7, 4, 56, 34]
#   Maximum element of the given list is :  56